meshmode/discretization/visualization.py

```diff
@@ -423,46 +423,94 @@ def _vtk_lagrange_connectivity(self):
         assert self.is_equidistant
         return VTKLagrangeConnectivity(self.connection)
 
-    def write_high_order_vtk_file(self, file_name, names_and_fields,
-            compressor=None, real_only=False, overwrite=False):
-        """Writes arbitrary order Lagrange VTK elements. These elements are
-        described in `this blog post <https://blog.kitware.com/modeling-arbitrary-order-lagrange-finite-elements-in-the-visualization-toolkit/>`_
-        and are available in VTK 8.1 and newer.
-        """     # noqa
-        if not self.is_equidistant:
-            raise RuntimeError("Cannot visualize high-order Lagrange elements "
-                    "using a non-equidistant visualizer. "
-                    "Call 'make_visualizer' with 'force_equidistant=True'.")
-
-        self._write_vtk_file(file_name, names_and_fields,
-                connectivity=self._vtk_lagrange_connectivity,
-                compressor=compressor,
-                real_only=real_only,
-                overwrite=overwrite)
-
-    def write_vtk_file(self, file_name, names_and_fields,
+    def write_par_vtk_file(self, mpi_comm, file_name_pattern, names_and_fields,
                 compressor=None, real_only=False,
-                overwrite=False, pvtu_filename=None,
-                part_nameformat=None, nranks=1):
-        r"""
-        :arg par_nameformat: If pvtu_filename and par_nameformat are given
-            by the caller (and non-empty), they are expected to be PVTU file
-            name to be written and a format string used to generate a rank-specific
-            filename for the part source in the PVTU format, respectively.
-            When used by a parallel application, only one rank should write
-            the PVTU file.
+                overwrite=False, use_high_order=None):
+        r"""A convenience wrapper around :meth:`write_vtk_file` for
+        distributed-memory visualization. The file name of the manife
```

Comment:
```suggestion
        distributed-memory visualization.
```
? Otherwise there's something missing.

Reply:
ed96160